src/java.base/share/native/libjava/System.c

```diff
@@ -206,6 +206,8 @@ Java_java_lang_System_initProperties(JNIEnv *env, jclass cla, jobject props)
 
     PUTPROP(props, "java.specification.version",
             VERSION_SPECIFICATION);
+    PUTPROP(props, "java.specification.maintenance.version",
+                    "2");
```

Comment:
Same here, please check rows above and below.

Reply:
Thank you for catching that, fixed the indentation for this and your previous comments.

Reply:
Great, thanks for updating. Looks nice now. 😄